data/settings.json

```diff
@@ -63,32 +59,32 @@
     },
     "speakers": {
       "title": "Speakers",
-      "metaDescription": "Hear from the Googlers, Partners, and Guest Speakers who are building the future of cloud. Check back often as we add more speakers, including our customers and partners.",
-      "description": "Hear from the Googlers, Partners, and Guest Speakers who are building the future of cloud. Check back often as we add more speakers, including our customers and partners.",
+      "metaDescription": "Venha aprender com os nossos experientes speakers convidados e empresas parceiras . Continuaremos a adicionar mais speakers a lista durante a semana. Volte mais tarde para ver mais novidades.",
+      "description": "Venha aprender com os nossos experientes speakers convidados e empresas parceiras . Continuaremos a adicionar mais speakers a lista durante a semana. Volte mais tarde para ver mais novidades.",
```

Comment:
as @rosariopfernandes corrected the previous "metaDescription" key, I think you should correct this as well
"metaDescription": "Venha aprender com os nossos oradores experientes e empresas parceiras. Continuaremos a adicionar mais oradores à lista durante a semana. Volte mais tarde para ver mais novidades.",

Reply:
sure.